packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/quanticSearchBoxInput.js

```diff
@@ -65,50 +67,51 @@ export default class QuanticSearchBoxInput extends LightningElement {
    */
   @api suggestions = [];
   /**
-   * Returns and set the input value.
+   * The value of the input.
    * @api
-   * @type {string}
+   * @type {String}
    */
-  @api
-  get inputValue() {
-    return this.input.value;
-  }
-  set inputValue(newValue) {
-    this.input.value = newValue;
-  }
+  @api value;
```

Comment:
Still need to change this back to `inputValue` to not be a breaking change without justification.

Reply:
added back.